playwright.config.js

```diff
@@ -7,7 +7,7 @@ export default defineConfig({
   workers: process.env.CI ? 1 : undefined,
   expect: {
     toHaveScreenshot: {
-      threshold: 0.0, // We expect colors to be exact matches.
+      threshold: 0.01, // We expect colors to be near exact matches.
```

Comment:
I was getting issues with local not matching CI. 0.01 is still really precise. (The default is 0.2 and was too imprecise to detect colors changing)